plugin/gutentags.vim

```diff
@@ -111,17 +111,33 @@ endif
 
 " Gutentags Setup {{{
 
+" Autocommands for updating the tags on save.
+augroup gutentags_buffer
+    autocmd!
+    autocmd BufWritePost <buffer> call gutentags#write_triggered_update_tags()
+augroup end'
+
+function! s:lazy_setup()
+    if g:gutentags_generate_on_new
+                \ || g:gutentags_generate_on_missing
+                \ || (get(g:, 'gutentags_auto_set_tags', 1)
+                \     && index(g:gutentags_modules, 'ctags') != -1)
```

Comment:
Gutentags should be set up without checking for particular modules. It keeps a better separation of concerns, and prevent bugs for people who may (in the future, when it works :) ) only have the `cscope` module enabled, for instance.